### DISCUSSION TITLE
Asking community for help on establishing a long term incentivised liquidity pool for KTON/RING

### DISCUSSION THREAD

**Discussion by SasoLithops** [ktondao/KtonDAO#35] [Q&A]
Based on:

This poll: https://github.com/orgs/darwinia-network/discussions/1577#discussion-7095035
This reply: https://github.com/orgs/darwinia-network/discussions/1577#discussioncomment-11149843
This KTON DAO treasury account: https://explorer.darwinia.network/address/0x08837De0Ae21C270383D9F2de4DB03c7b1314632?tab=internal_txns
**_This ongoing debate_**: https://github.com/orgs/darwinia-network/discussions/1627
And this initial discussion: https://github.com/orgs/darwinia-network/discussions/1566#discussion-7052819

___________________________________________________________________________________________________________________________________________________

**IMPORTANT: At this time KTON stakers get 19000 RING while KTON DAO treasury gets 4650 RING every 6 hours.**

**_ALSO: We are working on building an Index token which will highly benefit KTON stakers and governance and Github participants(Alpha). Any opinion on how we could implement it in staking rewards or its fees to KTON stakers, LP providers is welcome(check link above about ongoing debate). Most important KTON will be part of index token._**

_________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________
___________________________________________________________________________________________________________________________________________________

## 1. The first thread is on ratio of rewards to treasury and LP providers

<html>
<body>
<html><head></head><body><p>When determining how to allocate the <strong>4,650 RING</strong> rewards that go to the <strong>KTON treasury</strong> every 6 hours, and also reward <strong>KTON/RING liquidity providers</strong>, it’s important to consider the <strong>strategic goals</strong> of the <strong>KTON DAO</strong>, as well as the need to <strong>incentivize liquidity</strong> and maintain a <strong>healthy treasury</strong>. The allocation should aim to balance both <strong>treasury growth</strong> and <strong>liquidity provisioning</strong> to ensure long-term sustainability of the KTON ecosystem.</p>
<h3><strong>1. Overview of the Allocation Problem</strong></h3>
<p>You have <strong>4,650 RING</strong> that needs to be split between:</p>
<ul>
<li><strong>KTON DAO Treasury</strong> (to support the DAO's future growth, operations, and governance)</li>
<li><strong>KTON/RING Liquidity Providers (LPs)</strong> (to ensure sufficient liquidity for trading between KTON and RING)</li>
</ul>
<p>Both of these are essential to the ecosystem, but their roles are different:</p>
<ul>
<li><strong>Treasury</strong>: This is crucial for the DAO’s operations, including funding for development, governance, and future initiatives.</li>
<li><strong>KTON/RING LPs</strong>: Liquidity providers ensure that there’s a healthy market for the KTON/RING trading pair, allowing users to trade the tokens without experiencing slippage or high fees.</li>
</ul>
<h3><strong>2. Suggested Allocation of 4,650 RING Rewards</strong></h3>
<h4><strong>A. Reward Allocation to KTON Treasury</strong>:</h4>
<p>The <strong>KTON treasury</strong> is crucial for funding the DAO’s <strong>long-term objectives</strong> (e.g., governance, project development, treasury reserves for strategic initiatives, etc.). A reasonable portion of the rewards should go here to ensure the DAO remains financially sustainable.</p>
<ul>
<li><strong>Suggested Allocation to KTON Treasury</strong>: <strong>50% - 60%</strong> of the <strong>4,650 RING</strong> rewards should go to the <strong>KTON DAO Treasury</strong>. This will help build the treasury and ensure that the DAO can continue to fund development, governance, and other critical functions.
<ul>
<li><strong>50% of 4,650 RING</strong> = <strong>2,325 RING</strong> to KTON Treasury</li>
<li><strong>60% of 4,650 RING</strong> = <strong>2,790 RING</strong> to KTON Treasury</li>
</ul>
</li>
</ul>
<p>This ensures that the treasury receives a <strong>substantial portion</strong> of the rewards, maintaining a solid financial foundation for the DAO.</p>
<h4><strong>B. Reward Allocation to KTON/RING Liquidity Providers (LPs)</strong>:</h4>
<p>In order to maintain a <strong>liquid and efficient KTON/RING trading pair</strong>, it’s important to incentivize liquidity providers. However, the rewards to LPs should be balanced so that <strong>enough liquidity</strong> is available without overly depleting the treasury.</p>
<ul>
<li><strong>Suggested Allocation to KTON/RING LPs</strong>: <strong>40% - 50%</strong> of the <strong>4,650 RING</strong> rewards should go to <strong>KTON/RING liquidity providers</strong>. This incentivizes liquidity provision without diminishing the treasury’s ability to fund key initiatives.
<ul>
<li><strong>40% of 4,650 RING</strong> = <strong>1,860 RING</strong> to KTON/RING LPs</li>
<li><strong>50% of 4,650 RING</strong> = <strong>2,325 RING</strong> to KTON/RING LPs</li>
</ul>
</li>
</ul>
<p>This allocation encourages <strong>liquidity provision</strong> without overburdening the treasury, and ensures that LPs are sufficiently incentivized to participate in the KTON/RING pool.</p>
<hr>
<h3><strong>3. Proposed Reward Distribution Summary:</strong></h3>

Allocation | Percentage | RING Amount
-- | -- | --
KTON DAO Treasury | 50% - 60% | 2,325 RING - 2,790 RING
KTON/RING Liquidity Providers | 40% - 50% | 1,860 RING - 2,325 RING

<h3><strong>4. Considerations for Both Allocation and Liquidity Providers:</strong></h3>
<h4><strong>A. KTON DAO Treasury Role</strong>:</h4>
<ul>
<li>The <strong>treasury</strong> funds important DAO operations such as <strong>development</strong>, <strong>marketing</strong>, and <strong>ecosystem growth</strong>.</li>
<li>A <strong>solid treasury</strong> also provides a buffer for unforeseen expenses and strategic investments (e.g., acquisitions, partnerships).</li>
<li>By ensuring the treasury gets a substantial portion of the rewards, you are supporting the <strong>long-term viability</strong> of the KTON ecosystem.</li>
</ul>
<h4><strong>B. KTON/RING Liquidity Providers Role</strong>:</h4>
<ul>
<li><strong>LPs are critical</strong> for ensuring that the KTON/RING pool is liquid, which in turn helps users trade between KTON and RING without high slippage or fees.</li>
<li>The <strong>more liquidity in the pool</strong>, the more attractive the KTON ecosystem becomes to traders, increasing demand for both KTON and RING.</li>
<li><strong>LPs face impermanent loss risks</strong>, so the reward for providing liquidity should be attractive enough to compensate for that risk.</li>
<li>It may also be worth considering whether to <strong>split the LP rewards</strong> between <strong>RING</strong> and <strong>KTON</strong> as dual incentives, with <strong>RING DAO</strong> providing <strong>RING</strong> rewards and <strong>KTON DAO</strong> providing <strong>KTON</strong> rewards. This would provide LPs with <strong>two tokens</strong> instead of one, enhancing the attractiveness of providing liquidity.</li>
</ul>
<h4><strong>C. Balancing Liquidity and Treasury Health</strong>:</h4>
<ul>
<li>The <strong>treasury</strong> must remain healthy to ensure that it can support the ecosystem, including <strong>staking rewards</strong>, <strong>future DAO governance</strong>, and <strong>strategic investments</strong>.</li>
<li><strong>Liquidity providers</strong>, on the other hand, are crucial for ensuring <strong>market efficiency</strong> for both tokens. If liquidity is insufficient, the trading experience could suffer, which could negatively affect the ecosystem’s health.</li>
<li>A <strong>balanced approach</strong> is important to avoid putting too much strain on either the treasury or liquidity provisioning.</li>
</ul>
<hr>
<h3><strong>5. When to Adjust the Ratios</strong>:</h3>
<p>The exact ratio between <strong>treasury rewards</strong> and <strong>LP rewards</strong> can be adjusted over time based on certain conditions:</p>
<ul>
<li>If there is a <strong>drop in KTON liquidity</strong> or an increase in <strong>demand for KTON/RING trading</strong>, you may choose to <strong>increase the LP rewards</strong> to incentivize more liquidity provision.</li>
<li>If the <strong>KTON DAO treasury</strong> needs more funds for a particular initiative (such as a <strong>large development project</strong>), you can increase the treasury’s share of the rewards temporarily.</li>
</ul>
<p>Likewise, if the <strong>LP rewards are overly diluting</strong> the treasury or creating significant supply inflation, you may adjust the reward allocation to give a greater share to the treasury.</p>
<h3><strong>6. Additional Strategies for Reward Allocation</strong>:</h3>
<p>You can also consider implementing <strong>dynamic reward systems</strong> based on certain KPIs (Key Performance Indicators), such as:</p>
<ul>
<li><strong>Liquidity depth</strong>: If the KTON/RING pool has insufficient liquidity, the reward allocation could be skewed toward <strong>LPs</strong> until liquidity levels improve.</li>
<li><strong>Treasury balance</strong>: If the treasury is sufficiently funded and the DAO is focused on liquidity growth, more rewards could be allocated to <strong>LPs</strong>.</li>
<li><strong>Market conditions</strong>: If the market is under stress, the DAO might allocate more rewards to <strong>LPs</strong> to maintain liquidity in the ecosystem.</li>
</ul>
<hr>
<h3><strong>Final Recommendation:</strong></h3>
<p>Based on the current reward distribution, the ideal split for the <strong>4,650 RING</strong> rewards should be:</p>
<ul>
<li><strong>50% - 60%</strong> to the <strong>KTON DAO Treasury</strong> (2,325 RING - 2,790 RING).</li>
<li><strong>40% - 50%</strong> to <strong>KTON/RING Liquidity Providers</strong> (1,860 RING - 2,325 RING).</li>
</ul>
<p>This allocation ensures that both the <strong>treasury</strong> and <strong>liquidity providers</strong> are well-supported, and that the ecosystem can continue to grow sustainably. Adjustments to the reward structure can be made over time based on evolving market conditions, DAO needs, and liquidity demands.</p></body></html>
</body>
</html>

______________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________
______________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________

## 2. The second thread is should we incentivise or provide liquidity

Deciding whether to provide liquidity yourself or incentivize others to provide liquidity is a key decision for any decentralized project, and it depends on a variety of factors. Let’s break it down:

#### **Advantages:**

1. **Full Control**:
   - By providing liquidity yourself, you have **complete control** over the amount of liquidity in the pool. You can manage the risk and decide how much capital to allocate to liquidity provision.
   - You can adjust your position based on your assessment of the market or your project’s needs.

2. **Fee Earnings**:
   - As a liquidity provider (LP), you earn transaction fees based on the volume of trades in the liquidity pool. If you have a substantial portion of the liquidity, this can be a good source of passive income.
   
3. **Stabilizing the Pool**:
   - If the market is small or there are few liquidity providers, your contribution can help **stabilize the market** and reduce **slippage** for traders. This can help improve the **price efficiency** and trading experience, which may attract more users to your ecosystem.

#### **Disadvantages:**

1. **Capital Commitment**:
   - **Providing liquidity yourself** can tie up a significant amount of your capital. If you’re providing liquidity in a **KTON/RING pair, for example**, you’ll need to provide both tokens, and this could be risky if either token’s price fluctuates significantly.
   - This could be a **capital-intensive activity** unless you’re able to deploy a relatively small amount of capital.

2. **Impermanent Loss**:
   - When you provide liquidity in a decentralized exchange (DEX) like Uniswap or PancakeSwap, you’re exposed to **impermanent loss**. This occurs when the price of one token in the pair moves significantly relative to the other, leading to a potential **loss in value** of your liquidity share. However, transaction fees can help offset this risk if there’s enough trading volume.
   
3. **Opportunity Cost**:
   - By committing capital to liquidity provision, you may miss out on other investment opportunities. If the rewards from liquidity provision (fees) are not sufficient, it might not be the best use of your capital.

---

#### **Advantages:**

1. **Reduced Capital Commitment**:
   - **Incentivizing liquidity providers (LPs)** allows you to **leverage external capital** without tying up your own. This reduces the amount of your project’s capital that is directly exposed to the market.
   - You can attract large liquidity providers (LPs) who have **more capital** and can provide more liquidity than you alone, resulting in a deeper pool.

2. **Faster Liquidity Growth**:
   - Incentives (e.g., liquidity mining rewards) can attract a large number of LPs quickly, providing **rapid growth in liquidity**. This can lead to better price stability, lower slippage, and a more efficient market, especially in early stages when liquidity is crucial for the success of the market.
   
3. **Better Risk Management**:
   - If liquidity is provided by others, you can **minimize your exposure** to impermanent loss and the risks associated with market volatility. You can also more easily adjust the incentives (rewards) to match market conditions or balance supply and demand.
   
4. **Community Building and Engagement**:
   - Offering rewards to LPs not only provides liquidity but also helps build a **community of participants** who are incentivized to stay engaged with your project. This increases the **decentralization** of your ecosystem and can build long-term support.
   - Community members who provide liquidity are more likely to **stake tokens**, participate in governance, and **help promote the ecosystem**.

#### **Disadvantages:**

1. **Cost of Incentives**:
   - The **main downside** of incentivizing LPs is that it comes at a **cost** to the project. Rewards need to be provided to LPs, typically in the form of the native token or another valuable asset. This could lead to **inflation** if not managed carefully.
   - If the incentives are too high, it can result in **unsustainable token emissions** and **dilution of value** for existing token holders.

2. **Vulnerability to Farming**:
   - If the rewards are too attractive or short-term focused, liquidity providers might engage in **“liquidity farming”**—providing liquidity only to capture rewards, and then exiting as soon as the rewards dry up. This can lead to **liquidity volatility** and poor price stability after the farming period ends.

3. **Dependency on External Liquidity Providers**:
   - Relying on external liquidity providers means you have less **control** over the liquidity supply. If external LPs decide to withdraw liquidity (e.g., due to changes in rewards or market conditions), it could cause **slippage** or **price instability** in the pool.
   - There's also the risk that LPs might not fully align with the project’s long-term vision, as their incentives are often driven by **short-term rewards**.

---

#### **When to Provide Liquidity Yourself:**

- **Small Market or New Project**: If you’re launching a new token and liquidity is thin, providing liquidity yourself can help **bootstrap the market** and attract users. Without liquidity, the token won’t have much utility, so having enough liquidity early on is crucial.
- **Limited Resources for Incentives**: If the project doesn’t have sufficient capital to fund **liquidity incentives** long-term or doesn’t want to rely on external liquidity providers, providing liquidity yourself can be a viable option.
- **Active Participation**: If you're heavily invested in the project and want to **ensure price stability** or are looking to **manage risk**, providing liquidity yourself allows you to directly control the liquidity in the market.
  
#### **When to Incentivize Liquidity Providers:**

- **Scaling the Market Quickly**: If you need to quickly increase liquidity and get a **deeper pool**, incentivizing external liquidity providers is often the best choice. Offering liquidity mining rewards (in the form of RING, KTON, or other assets) can attract more LPs faster than doing it on your own.
- **Sustaining Long-Term Liquidity**: Incentivizing liquidity providers helps you ensure there’s **consistent and long-term liquidity** in the pool, which is especially important as your market grows.
- **Leveraging Community**: If your project is trying to **build a decentralized community** or **engage users** in governance and decision-making, rewarding LPs with incentives can foster a sense of ownership and participation in the ecosystem.
- **Cost Management**: If your treasury is not large enough to commit significant capital for liquidity provision, incentivizing others with rewards is a more **cost-effective strategy**.

---

### **Conclusion:**

- **Provide liquidity yourself** if:
  - You want full control over the liquidity.
  - You're in the early stages and need to bootstrap the market.
  - You have sufficient capital and want to directly support price stability.
  
- **Incentivize liquidity providers** if:
  - You need to scale liquidity quickly and sustainably.
  - You want to reduce your personal capital commitment and rely on the community.
  - You're aiming to decentralize liquidity and build long-term engagement within your ecosystem.

In most cases, **a combination of both strategies** is ideal. You could start by providing liquidity yourself (especially in the early stages) to **bootstrap the market** and then transition to incentivizing external LPs to **scale liquidity** over time. By managing both approaches carefully, you can **optimize liquidity** for both stability and growth.

______________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________
______________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________

## 3.The third is should both RING DAO and KTON DAO provide liquidity incentives, community already voted mostly yes on poll(link in the begining above)

The idea of having both the **RING DAO** and **KTON DAO** reward **KTON/RING liquidity providers** is an interesting one and could have several benefits, but it also requires careful consideration of how it would align with both DAOs' goals, reward structures, and long-term sustainability.

### **Pros of Both DAOs Rewarding KTON/RING Liquidity Providers**

1. **Double Incentives (Attract More Liquidity)**:
   - If both DAOs (RING DAO and KTON DAO) reward liquidity providers in the **KTON/RING pair**, it would create **double rewards** for those providing liquidity to this pair. This could attract more liquidity to the pool and increase the overall depth and stability of the KTON/RING market.
   - **RING DAO** could offer rewards in **RING**, while **KTON DAO** could reward liquidity providers with **KTON**. The dual reward structure would make the liquidity provision more attractive because providers would earn both tokens (RING and KTON), diversifying their holdings and providing incentives to maintain liquidity in the pool.

2. **Strengthening the Ecosystem**:
   - Offering incentives from both DAOs would align both tokens (RING and KTON) in terms of liquidity and use within the ecosystem, driving **more activity** around the KTON/RING pair. This helps **grow the KTON and RING markets** by ensuring both tokens have sufficient liquidity for trading and future use cases within the ecosystem.
   - **Increased liquidity** would reduce slippage in trades between RING and KTON, making the ecosystem more efficient, which is beneficial for both DAOs and the broader community.

3. **Cross-Incentivization**:
   - By involving both DAOs, you create a form of **cross-incentivization** where the success of one DAO helps the other. For example, **KTON DAO** rewards liquidity providers in KTON, potentially increasing the demand for KTON (driving up its price). Meanwhile, **RING DAO** can reward liquidity providers in RING, which increases the demand for RING.
   - This cross-rewarding could create a **virtuous cycle** where liquidity provision in the KTON/RING pair becomes a highly attractive strategy for users, leading to more trading, better price discovery, and deeper liquidity.

4. **Attracting More Capital**:
   - Providing rewards from both DAOs could **attract larger liquidity providers**, including institutional players, who prefer dual incentives because they get exposure to both assets and can hedge some risk between the two.
   - This would also **improve the overall capital efficiency** of the liquidity pool, as large liquidity providers are often looking for ways to generate passive income in DeFi, and dual rewards make it more enticing.

---

### **Challenges and Considerations**

1. **Reward Inflation**:
   - One concern is the potential **inflationary pressure** on both the RING and KTON tokens. Offering rewards from both DAOs could lead to **excessive token issuance**, which might undermine the value of these tokens over time if not carefully managed.
   - To avoid this, both DAOs would need to ensure that the reward distribution is sustainable and does not lead to excessive supply expansion. This can be addressed by implementing **caps** on the reward distribution or **adjusting rewards over time** to ensure they remain in line with the actual liquidity being provided.

2. **Alignment of DAO Objectives**:
   - While both DAOs might benefit from increased liquidity in the KTON/RING pool, it’s essential that the **reward structure is aligned** with the goals of each DAO. For instance:
     - **KTON DAO** might want to **increase demand** for KTON and strengthen its token’s value.
     - **RING DAO** might be more focused on expanding the use of RING in decentralized finance and driving **network growth**.
   - If the rewards aren't balanced or if they serve conflicting goals, it could create **misaligned incentives** that might lead to issues like too much liquidity in the KTON/RING pool and not enough demand for either token elsewhere in the ecosystem.

3. **Complexity of Managing Dual Incentives**:
   - The administration and **tracking of dual rewards** can become complex. The DAOs will need to have a clear mechanism in place to **distribute rewards** accurately and **track liquidity provider contributions**. For example, how would rewards be split between KTON and RING, and what metric would be used to measure liquidity provision (e.g., weighted by the amount of liquidity provided, or by trading volume)?
   - This also creates potential for governance complexity, where decisions about reward allocation or adjustments to reward programs may need to be coordinated between both DAOs.

4. **Risk of Imbalance in Liquidity Pool**:
   - If one token (e.g., KTON) has significantly higher rewards than the other, it could lead to an **imbalance** in the liquidity pool, with more of one token being supplied than the other. For instance, if KTON DAO offers much higher rewards in KTON than RING DAO offers in RING, liquidity providers might disproportionately provide KTON rather than RING, leading to an **overexposure** to KTON in the KTON/RING pool.
   - This imbalance could cause **unstable pricing** between the two tokens and reduce the pool’s ability to function effectively.

---

### **Proposed Reward Allocation Strategy**

1. **Reward Caps**: 
   - Each DAO could offer rewards **proportional to their supply** and **market cap**, to avoid excessively inflating the supply of either RING or KTON. For example, each DAO could reward liquidity providers based on a fixed percentage of total liquidity in the pool, ensuring that the rewards are balanced and sustainable.
   
2. **Weighting Based on Token Contribution**:
   - The rewards could be **weighted** by the amount of liquidity each token contributes to the pool. For example, if liquidity providers are supplying more RING than KTON, the RING DAO might issue a larger reward to balance the pool.
   - This can be structured in a way that avoids **over-concentration of rewards** in one token, while still encouraging liquidity providers to provide an appropriate mix of RING and KTON.

3. **Adjustable Reward Mechanisms**:
   - To ensure sustainability, both DAOs could introduce an **adjustable reward mechanism**, where rewards are not static but are adjusted based on the total liquidity in the pool or the **token price**. If RING or KTON prices fall, rewards could be dynamically adjusted to incentivize more liquidity provision. Similarly, if liquidity in the pool becomes too high, rewards could be reduced to maintain balance.

4. **Time-Locked or Vesting Period**:
   - To prevent liquidity providers from exiting the pool too quickly, rewards could have a **vesting period** or be **time-locked**. This ensures that liquidity providers are incentivized to remain in the pool for longer periods, helping stabilize liquidity over time.

5. **Governance Involvement**:
   - Both DAOs could involve their communities in **governance decisions** around the reward structure. This would help ensure that the reward program is constantly evolving to meet the needs of liquidity providers while staying in line with the long-term vision of each DAO.

---

### **Conclusion: Should Both RING DAO and KTON DAO Reward KTON/RING Liquidity Providers?**

Yes, **rewarding KTON/RING liquidity providers from both DAOs (RING DAO and KTON DAO)** can be a **strategically sound approach** if done carefully. Here’s why:

1. **Increased Liquidity**: Dual rewards (RING and KTON) would likely **increase liquidity** in the KTON/RING pair, leading to better price stability, lower slippage, and more robust trading activity within the ecosystem.
2. **Stronger Ecosystem**: By rewarding liquidity providers, both DAOs can **strengthen their ecosystems**, supporting their tokens' value, utility, and market presence.
3. **Cross-incentivization**: This approach can create **synergy** between both DAOs, aligning their goals and increasing the overall liquidity in the combined market.

### **Caution**:
- Both DAOs need to carefully **monitor the reward distribution** to avoid **inflationary risks** or **imbalances** in liquidity, which could undermine the value of either token.
- Governance should be used to **regularly adjust rewards** and ensure the system remains sustainable over time.

If implemented thoughtfully, the rewards from both DAOs can serve to **attract more liquidity** while **rewarding participants** in a way that benefits both RING and KTON holders, as well as the broader DeFi ecosystem.

**Comment by sonoferin2000** [comment]
I think it’s worth asking if it’s in the DAO’s long-term interest to keep paying out 40-50% of rewards just to "rent" liquidity. Incentives are great, but they shouldn’t have to be a permanent thing. They should be used to achieve a goal, not just handed out endlessly.

Do we have any data on how these incentives are actually helping? Other pools don’t get incentives and they’re still functional, but the KTON pool is incentivized and still pretty shallow. For example, buying just 10 KTON causes a 2.26% price impact. If we’re paying out incentives, we should see results. Right now, it doesn’t feel sustainable to keep paying like this without clear benefits.

Here’s another idea: Instead of constantly incentivizing liquidity providers, why don’t we use that 40-50% of rewards to buy KTON ourselves, bridge it to Darwinia, and add it to the liquidity pool? The DAO could provide the KTON, and RINGDAO could supply the RING. We could split the LP tokens equally and keep the funds within the ecosystem. Over time, we’d build up a strong liquidity position that we don’t have to rent, and we’d free up resources for other projects.

If we ever need extra liquidity, we can still use incentives as a tool, but they don’t need to be a permanent fixture. It can’t be Christmas every day. Paying out rewards like this indefinitely isn’t viable, and I think we’d get way more value from building something sustainable instead.

**Comment by SasoLithops** [comment]
Very valid points @sonoferin2000. Problem is we have very little ammount in treasury. @hackfisher are there any other tokens in KTON DAO treasury? I think maybe incentivise at start and with KTON buybacks later we could work towards providing some liquidity as KTON DAO to not be depandand on "renting" as you nicely put it.
Also another option could be use 20% for incentives and 20% for buying KTON and using it in liquidity. But we need RING DAO to help in providing Ring in that pool.

Now KTON pool is shallow as most of us (community) has pulled its liquidity out as IL would devour it. Incentives can cover that loss as fees don't accumulate enough extra income like in some huge ETH/USDT pools.

**Reply by hackfisher** [reply]
Hi,

I propose to keep things simple, and I agree with sonoferin2000’s opinion. My favorite proposal would look like this:
	1.	Allocate 50% of the RING inflow from the KtonDAO Treasury to improve liquidity.
	2.	How to improve liquidity? Take half of this RING to purchase KTON from the RING-KTON AMM, and pair it with the remaining half of the RING to form an LP, then add it to the AMM. This process can be automated and long-term, requiring little to no intervention over time. It would accumulate and grow instead of being a short-term incentive cost (which essentially becomes ineffective after the incentives are used up). This approach focuses on building cumulative liquidity rather than expenditure.
	3.	What if the allocated amount is not sufficient? I believe increasing the Treasury’s income ratio and reducing KTON staking rewards could be a solution. For example, increasing the share from 20% to 50% (I think this is acceptable).

**Reply by hackfisher** [reply]
And for the smart contract support, it may need some technical support from @hujw77 , but I thinks it deserves and we do not need to worried about liquidity incentive next time(only curate some ratio parameters), so we can focus on building new arms.

**Reply by SasoLithops** [reply]
Hi,

Although I am starting to agree with the points you both make, I have two issues with this. First one is why should KTON DAO alone take care of the liquidity that benefits also RING DAO as sellers of KTON are also users of this LP pool. I understand this complicates things, but maybe Ring Dao can offer some extra Ring will come to why I think that.
Which brings me to a second point that hurting stakers to lose another 37,8% of ammount they receive now, to bring the 50% treasury-50% stakerswould trigger a massive sell out of KTON. The KTON stakers already let go of 20% of rewards to fund the treasury. While KTON and RING were part of Darwinia from start somehow all funds that were inherited went to RING DAO. I think funds should come from other way of funding by not hurting KTON stakers.

**Reply by hackfisher** [reply]
The monetary policy of each DAO token should be determined by the holders of that token. Therefore, the issuance and distribution of RING should be decided by RingDAO, and the issuance and distribution of KTON should be decided by KtonDAO. I believe this is natural, and under no circumstances should monetary policy be dictated by another organization. This would inevitably lead to problems, as other DAOs, lacking core interest in the matter, will always tend to favor allocating more to themselves if given the opportunity.

As for the third point, I think it’s optional and can be dropped. The choice of specific proportions should be left to a vote, as that would truly reflect the opinions of KTON holders. (For instance, not all KTON voters may care about short-term RING incentives; they might instead favor the management and investment of the Treasury.)

**Reply by SasoLithops** [reply]
I respect your opinion.
It wasn't my intention to start drama with Ring DAO I just feel that KTON DAO drew the shorter straw and will struggle to collect decent ammount of treasury to get going. My idea was we would definitely propose to Ring DAO to get some part of treasury in RING to accelerate our treasury and provide decent ammount in Ring/KTON LP pool. Nothing would be for self gains and we would keep it at highest transparency with sharring address to track funds and transactions. 

But that is only my opinion. I am not burdened by this and will move pass this and leave it in the past if voters decide otherwise. Bad drama is the worst thing that could happen to us.

**Reply by hackfisher** [reply]
RingDAO has to maintain a certain level of inflation to cover Darwinia’s minimum security and operational costs (such as consensus and collator expenses). However, KtonDAO doesn’t face such burdens or pressure—it has a smaller supply and already generates income.

As for growth, I believe adopting a lightweight approach to provide positive external value and services offers greater potential and deserves more of our attention.

**Reply by SasoLithops** [reply]
All true and agree. But all these come with costs which are impossible to cover at this point. Maybe focusing alone on Index should be priority then and let liquidity and BTC come at later stage when more income is generated. Let's see what others have to say. But we could start with KTON buybacks to bring some KTON into treasury.

**Reply by sonoferin2000** [reply]
I think the 50% idea makes a lot of sense. Honestly, we should’ve started funding much earlier. Sure, 50% feels like a big number—it is—but it gives us a real chance to make meaningful progress. We can tackle the liquidity issue while also addressing the KTON buybacks Hackfisher mentioned in the thread, using this proposal to hit both goals. We can always lower the funding rate later, but right now, the DAO really needs these resources to get things moving and set up the right systems. I know everyone wants to maximize their rewards—I do too—but if we keep the current funding rate, it’s going to take forever to make any real progress. Not everyone might agree, but I hope they can see the logic in starting strong and building momentum.

**Reply by SasoLithops** [reply]
I remember we had similar discussion about RING token and took a lot of heat from stakers for it. I personaly don't mind 50% going to treasury, if community understands why we doing this and agrees. Maybe we could put a vote on this with explanation why we doing this of course. I think best in TG where most of KTON stakers are. 
I understand concept of long term and healthy and full treasury = healthy project. My bigger concern with treasury is that we only depend on RING. With KTON buybacks that will start to change.

**Reply by sonoferin2000** [reply]
im not opposed to doing a poll to gauge sentiment. would be nice to hear what people have to say,but would be more beneficial if we all could have actual discussion though, rather then just a yes or no. think here would be the best place for that. we can continue at the current funding rate, but its going to take a long time to accomplish anything at that rate. Im couriouse to know what view others have on this.

**Reply by sonoferin2000** [reply]
> RingDAO has to maintain a certain level of inflation to cover Darwinia’s minimum security and operational costs (such as consensus and collator expenses). However, KtonDAO doesn’t face such burdens or pressure—it has a smaller supply and already generates income.
> 
> As for growth, I believe adopting a lightweight approach to provide positive external value and services offers greater potential and deserves more of our attention.

can you give example of what you mean by a lightweight approach?

**Reply by SasoLithops** [reply]
@sonoferin2000 Agree it would be much more beneficial to have a discussion here where percentages can be further discussed... then just YES or NO. The problem is that it will take time for community to join us here. I am getting some small feedback from DMs as well.  

We could do it like in this example:

**YES**, I am willing to give up on some rewards, because I want our treasury to grow faster so we can develop more sooner.
**NO,** I am not willing to give up on my rewards. Let treasury grow at slow pace we are in no hurry to introduce changes.
**ABSTAIN,** I am not familiar with changes and trust community to vote on this.

Or something like COSMOS governance does by explaining each option:

### **Vote options**

The following items summarize the voting options and what they mean for this proposal.


**Yes**

You agree that this proposal was created by the DAO DAO team, and you wish to allow our wallet (cosmos1559zgk3mxm00qtr0zu2x5n4rh5vw704qaqj6ap) to deploy the DAO DAO CosmWasm smart contracts on the Cosmos Hub.


**No**

You do not wish to authorize our wallet (cosmos1559zgk3mxm00qtr0zu2x5n4rh5vw704qaqj6ap) to deploy the DAO DAO CosmWasm smart contracts on the Cosmos Hub, or you believe this proposal or wallet to not belong to the DAO DAO team.


**Abstain**

You wish to contribute to the quorum but formally decline to vote either for or against the proposal.


**NoWithVeto**

You believe this proposal either (1) is deemed to be spam, i.e., irrelevant to the Cosmos Hub, (2) disproportionately infringes on minority interests, or (3) violates or encourages violation of the rules of engagement as currently set out by Cosmos Hub governance. If the number of NoWithVeto votes is greater than 1/3 of total votes, the proposal is rejected and the deposit is burned.

**Reply by sonoferin2000** [reply]
doesnt hurt, but i would also include a link back to here, so if they can see what is being discussed, and if they  want to voice anything, they can. Im not opposed to it.

**Reply by SasoLithops** [reply]
Very valid point. Abslutley agree on link being present.

**Comment by 2xJDubs** [comment]
I know I am a bit late to the conversation but I agree with most of the discussion that we should look at a 50/50 split of the treasury funds with 50% used for supporting KTON liquidity but I have a different idea of how to get there.  Currently RingDAO treasury holds around 779 KTON.

https://darwinia.subscan.io/account/0x6d6f646c64612f74727372790000000000000000

This KTON is not being used by the RingDAO treasury and could be used by the KtonDAO treasury to pair with the 50% of the income to the DAO to add to the KTON/RING LP pool.  Now this can't be done all at once as the KtonDAO does not have enough RING in the treasury to pair but can be done over time in maybe bunches of 50 KTON whenever there is enough RING to pair with 50 KTON without draining the treasury.

I prefer this idea to @hackfisher idea of buying KTON from the pool and the resupplying it to the pool as I feel this does not deepen liquidity but maintains it where it is and just shifts the owner of the liquidity.

**Reply by SasoLithops** [reply]
I like this idea too. Should we move this forward and propose first to RING DAO for the KTON to be used in liquidity? It suits them as well as it would be RING paired. Please react with thumb up or thumb down if we continue with this. Of course welcome to leave comment as well if you think more debate on this needed.

@hackfisher @2xJDubs @sonoferin2000 @hujw77

**Comment by SasoLithops** [comment]
Does it make sense to all of us starting this liquidity pool on Base chain, so it would be easy for KTON and RING to be included in Index (we are trying to set it on Base) as well?

**Comment by SasoLithops** [comment]
https://github.com/orgs/ringecosystem/discussions/18
